wb/main/jobs/datasets/convert_dataset_job.py

```diff
@@ -87,14 +87,16 @@ def run_conversion(self, original_dataset: DatasetsModel,
                        dataset_format: DatasetTypesEnum):
         tool = DatumaroTool()
         tool.set_mode(DatumaroModesEnum.convert)
-        tool.set_input_output_paths(original_dataset.path, result_dataset.path)
+        tool.set_path('input-path', original_dataset.path)
+        tool.set_path('output-dir', result_dataset.path)
         tool.set_conversion(dataset_format, self._format_conversion_map[dataset_format])
-        tool.enable_image_copy()
+        tool.set_flag('')
```

Comment:
It is absolutely not understandable.  Please create a specific method to add `--`